src/main/java/com/google/cloud/tools/gradle/appengine/core/AppEngineCorePluginConfiguration.java

```diff
@@ -95,23 +95,29 @@ public void configureCoreProperties(
 
   private void configureFactories() {
     project.afterEvaluate(
-        project -> {
+        projectAfterEvaluated -> {
           try {
             if (toolsExtension.getCloudSdkHome() == null) {
               managedCloudSdk =
                   new ManagedCloudSdkFactory(toolsExtension.getCloudSdkVersion()).newManagedSdk();
               toolsExtension.setCloudSdkHome(managedCloudSdk.getSdkHome().toFile());
             }
-          } catch (UnsupportedOsException | BadCloudSdkVersionException ex) {
-            throw new GradleException("Configuring... ", ex);
+          } catch (UnsupportedOsException ex) {
+            throw new RuntimeException(ex.getMessage(), ex);
+          } catch (BadCloudSdkVersionException ex) {
+            throw new RuntimeException(
+                "Failed to auto-configure Cloud Sdk at version = '"
+                    + toolsExtension.getCloudSdkVersion()
+                    + "': "
+                    + ex.getMessage());
```

Comment:
Should also pass `ex` as the cause parameter, right?

Reply:
oh yeah.